NVIDIA-Driver/install.bash

```diff
@@ -64,7 +115,18 @@ echo -e "\e[32m The version of the driver is \e[33m""$([[ "$INSTALLER" =~ ^.*\-(
 echo "${BASH_REMATCH[1]}")\e[m"
 read -rp "Press any key to continue ... " -n1 -s
 echo
-if ! sudo sh "$INSTALLER" \
+# We only extract, otherwise ENV variables don't seem to affect the installer
+# This is also faster if the user wants to repete the installation
+if [ ! -d ./${DRIVER}]
+if ! sudo sh "$INSTALLER" --extract-only; then
+  echo -e "\e[31m Installation failed! Aborting...\e[m"
+  exit 1
+fi
```

Comment:
Two problems.

First, the syntax is incorrect
```bash
if [ ! -d ./${DRIVER} ]; then
  if ! sudo sh "$INSTALLER" --extract-only; then
    echo -e "\e[31m Installation failed! Aborting...\e[m"
    exit 1
  fi
fi
```

Second, according to this [post](https://forums.developer.nvidia.com/t/clear-linux-error-unable-to-load-the-nvidia-drm-kernel-module-gtx-970/124836/517302), setting `CONFIG_SECTION_MISMATCH_WARN_ONLY=y` should be sufficient and I don't understand why it doesn't work in your case.

Though extracting saves time, I don't think it makes a significant difference.

So unless it's made sure that we have to extract this, I prefer not.

Reply:
I added extraction because in a previous version of the driver I also had to apply some corrections to the files inside the driver. Now I don't need it anymore, but I still extract since OS updates are much more frequent than driver updates, and this saves up some time (especially on slow drives).

Reply:
Since this is not essential, I prefer to delete this. The extraction doesn't take much time in any case.